pkg/multus/multus.go

```diff
@@ -503,6 +504,22 @@ func cmdPluginErr(k8sArgs *types.K8sArgs, confName string, format string, args .
 	return logging.Errorf(msg+format, args...)
 }
 
+func isCriticalRequestRetriable(err error) bool {
+	logging.Debugf("isCriticalRequestRetriable: %v", err)
+	errorTypesAllowingRetry := []func(error) bool{
+		errors.IsServiceUnavailable, errors.IsInternalError, k8snet.IsConnectionReset, k8snet.IsConnectionRefused}
+	for _, f := range errorTypesAllowingRetry {
+		if f(err) {
+			return true
+		}
+	}
+	return false
+}
+
+func generateRetryBackoff(interval time.Duration, timeout time.Duration) wait.Backoff {
+	return wait.Backoff{Duration: interval, Factor: 2, Jitter: 0.1, Steps: 5, Cap: timeout}
```

Comment:
Can we make 1 item for 1 line (for clear visibility), as

`
        return wait.Backoff{
                Duration: interval,
                Factor: 2,
                Jitter: 0.1,
                Steps: 5,
                Cap: timeout
        }
`